var/spack/repos/builtin/packages/llvm-openmp/package.py

```diff
@@ -76,10 +93,31 @@ def patch(self):
             os.rename(cmake_mod_dir, os.path.join(self.stage.path, "cmake"))
 
     def cmake_args(self):
-        cmake_args = []
+
+        # Disable LIBOMP_INSTALL_ALIASES, otherwise the library is installed as
+        # libgomp alias which can conflict with GCC's libgomp.
+        cmake_args = ["-DLIBOMP_INSTALL_ALIASES=OFF"]
+        cmake_args.append("-DOPENMP_STANDALONE_BUILD=1")
         # Add optional support for both Intel and gcc compilers
         if self.spec.satisfies("+multicompat"):
             cmake_args.append("-DKMP_GOMP_COMPAT=1")
+
+        # Build llvm-openmp-ompt as a stand alone library
+        # CMAKE rpath variable prevents standalone error
+        # where this package wants the llvm tools path
+        if "+standalone" in self.spec:
+            cmake_args.extend(
+                [
+                    "-DLIBOMP_STANDALONE_BUILD=true",
```

Comment:
It looks like as if `LIBOMP_STANDALONE_BUILD`is and older or internal libomp variable and OPENMP_STANDALONE_BUILD is the externally settable variable:
https://gitlab.inria.fr/openmp/libkomp/blob/8668a01effa639990e3467c5b4ccf50fc88ad82e/runtime/CMakeLists.txt#L19
https://cgit.geodns-americas.gentoo.org/fork/llvm-project.git/diff/?h=gentoo-15.0.3&id=3e921d3c52542b3284c4bd0ef29a70511d38c79c
```suggestion
                    "-DOPENMP_STANDALONE_BUILD=ON",
```

Reply:
I changed to OPENMP_STANDALONE_BUILD.